README.md

```diff
@@ -36,6 +36,29 @@ The script is distributed via NPM, but doesn't actually require Node.js beyond t
 1. Run `chmod a+x cf-blue-green`.
 1. Move the file somewhere in your PATH.
 
+### Using with Travis
+
+Travis supports [continuous deployment](http://docs.travis-ci.com/user/deployment/), which will automatically deploy your application after its tests pass on a specified branch. To use `cf-blue-green` with Travis, you need to use a [script provider](http://docs.travis-ci.com/user/deployment/script/) instead of the default Cloud Foundry provider. Your Cloud Foundry settings are read from environment variables.
+
+Set up continuous deployment with the following settings in your `.travis.yml` file:
+
+```yml
+before_install: npm install -g cf-blue-green
```

Comment:
I updated it to run at `before_deploy`. It can't actually be in the script, because it has to run in order for the script to exist. Also added `sudo: true`, which looks like is required for now with Travis's new containers.